tests/roles/backend_services/templates/openstack_version.j2

```diff
@@ -0,0 +1,74 @@
+    edpmLogrotateCrondImage: {{ container_registry }}/{{ container_namespace }}/openstack-cron:{{ container_tag }}
+    edpmMultipathdImage: {{ container_registry }}/{{ container_namespace }}/openstack-multipathd:{{ container_tag }}
+    edpmNeutronMetadataAgentImage: {{ container_registry }}/{{ container_namespace }}/openstack-neutron-metadata-agent-ovn:{{ container_tag }}
+    edpmNeutronSriovAgentImage: {{ container_registry }}/{{ container_namespace }}/openstack-neutron-sriov-agent:{{ container_tag }}
+    edpmOvnBgpAgentImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-bgp-agent:{{ container_tag }}
+    glanceAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-glance-api:{{ container_tag }}
+    heatAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-heat-api:{{ container_tag }}
+    heatCfnapiImage: {{ container_registry }}/{{ container_namespace }}/openstack-heat-api-cfn:{{ container_tag }}
+    heatEngineImage: {{ container_registry }}/{{ container_namespace }}/openstack-heat-engine:{{ container_tag }}
+    horizonImage: {{ container_registry }}/{{ container_namespace }}/openstack-horizon:{{ container_tag }}
+    infraDnsmasqImage: {{ container_registry }}/{{ container_namespace }}/openstack-neutron-server:{{ container_tag }}
+    infraMemcachedImage: {{ container_registry }}/{{ container_namespace }}/openstack-memcached:{{ container_tag }}
+    ironicAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-ironic-api:{{ container_tag }}
+    ironicConductorImage: {{ container_registry }}/{{ container_namespace }}/openstack-ironic-conductor:{{ container_tag }}
+    ironicInspectorImage: {{ container_registry }}/{{ container_namespace }}/openstack-ironic-inspector:{{ container_tag }}
+    ironicNeutronAgentImage: {{ container_registry }}/{{ container_namespace }}/openstack-ironic-neutron-agent:{{ container_tag }}
+    ironicPxeImage: {{ container_registry }}/{{ container_namespace }}/openstack-ironic-pxe:{{ container_tag }}
+    ironicPythonAgentImage: {{ container_registry }}/{{ container_namespace }}/ironic-python-agent:{{ container_tag }}
+    keystoneAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-keystone:{{ container_tag }}
+    manilaAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-manila-api:{{ container_tag }}
+    manilaSchedulerImage: {{ container_registry }}/{{ container_namespace }}/openstack-manila-scheduler:{{ container_tag }}
+    manilaShareImage: {{ container_registry }}/{{ container_namespace }}/openstack-manila-share:{{ container_tag }}
+    mariadbImage: {{ container_registry }}/{{ container_namespace }}/openstack-mariadb:{{ container_tag }}
+    neutronAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-neutron-server:{{ container_tag }}
+    novaAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-api:{{ container_tag }}
```

Comment:
apart from manila/cinder the rest are good e.g. https://logserver.rdoproject.org/95/52995/2/check/periodic-data-plane-adoption-osp-17-to-extracted-crc/f7c0a71/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/nova-api-0/nova-api-0-describe

```
    Image:         quay.rdoproject.org/podified-antelope-centos9/openstack-nova-api:d5d608a096dad615f5580e3efd2f218c
```